features/svg-deprecated.yml

```diff
@@ -1,7 +1,11 @@
-draft_date: 2024-10-085
-name: svg-deprecated
-description: Placeholder for deprecated or non-real features
+name: SVG (deprecated)
+description: SVG 2 deprecated or removed a significant number of features defined in SVG 1.1, including features related to fonts, alternate glyphs, the `xlink` namespace, and the `<cursor>` and `<tref>` elements.
 spec: https://svgwg.org/svg2-draft/
+discouraged:
+  according_to:
+    - https://svgwg.org/svg2-draft/changes.html#substantial
```

Comment:
Rather than link to the individual parts that deprecate each feature, this link enumerates them for us.